src/mlpack/methods/quic_svd/quic_svd_impl.hpp

```diff
@@ -42,10 +59,11 @@ inline QUIC_SVD::QUIC_SVD(
 
   // Use the ExtractSVD algorithm mentioned in the paper to extract the SVD of
   // the original dataset in the obtained subspace.
-  ExtractSVD(u, v, sigma);
+  ExtractSVD(dataset,u, v, sigma);
```

Comment:
```suggestion
  ExtractSVD(dataset, u, v, sigma);
```
Minor style fix. :+1: